.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -0,0 +1,50 @@
+parameters:
+  clusterName: ""
+  os: ""
+  datapath: ""
+  netpol: ""
+
+
+jobs:
+  - job: k8s_e2e
+    steps:
+      - task: AzureCLI@1
+        inputs:
+          azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+          scriptLocation: "inlineScript"
+          scriptType: "bash"
+          addSpnToEnvironment: true
+          inlineScript: |
+            set -ex
+            make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
+
+            # sig-release provides test suite tarball(s) per k8s release. Just need to provide k8s version "v1.xx.xx"
+            # pulling k8s version from AKS.
+            eval k8sVersion="v"$( az aks show -g ${{ parameters.clusterName }}-$(make revision) -n ${{ parameters.clusterName }}-$(make revision) --query "currentKubernetesVersion")
+            curl -L https://dl.k8s.io/$k8sVersion/kubernetes-test-linux-amd64.tar.gz -o ./kubernetes-test-linux-amd64.tar.gz
+
+            # https://github.com/kubernetes/sig-release/blob/master/release-engineering/artifacts.md#content-of-kubernetes-test-system-archtargz-on-example-of-kubernetes-test-linux-amd64targz-directories-removed-from-list
+            # explictly unzip and strip directories from ginkgo and e2e.test
+            tar -xvzf kubernetes-test-linux-amd64.tar.gz --strip-components=3 kubernetes/test/bin/ginkgo kubernetes/test/bin/e2e.test
+        displayName: "Setup Environment"
+        condition: always()
```

Comment:
Why do we want the condition as `always` here ?

Reply:
Originally, I was running multiple jobs to get additional parallelism. `always()` Ensured that I could re-run a failed e2e test suite. Now I am running in steps as the cluster could not support any more parallelism past what was specified in a single e2e test suite.

Reply:
So we can remove this ?